core/src/gas.ts

```diff
@@ -1,23 +1,25 @@
 import { ethers } from 'ethers';
+import memoizee from 'memoizee';
 import BigNumber from './bignumber';
 import { getNetworkConfigByType } from './constants/NETWORKS';
 import { ETH_NUMBER_OF_DIGITS } from './constants/UNITS';
-import memoizee from 'memoizee';
-
-const API_URL = 'https://ethgasstation.info/json/ethgasAPI.json';
-const TRANSACTION_SPEED = 'fast';
+import CHAINLINK_FAST_GAS_ABI from './abis/CHAINLINK_FAST_GAS.json';
+import getProvider from './provider';
 
-const GAS_CACHE = 10 * 1000;
+const CHAINLINK_FAST_GAS_ADDRESS = '0x169E633A2D1E6c10dD91238Ba11c4A708dfEF37C';
```

Comment:
Will this address never change or will we need to manually recode this every once in a while?

Reply:
Just like with other addresses, they can't change. The only possibility is that they would be discontinued, it's an interesting question how can we prepare for this situation, but kind of unrelated to the discussion (: